### ISSUE TITLE
write an upgrade guide to avoid missing steps

### ISSUE THREAD

**Issue by dasilvacontin** [janl/mustache.js#437]

**Comment by dasilvacontin** [comment]
I missed updating the `CHANGELOG` before publishing. Will update later.

**Comment by phillipj** [comment]
Could we enforce the correct procedure somehow? Docs are great, but there
will always be room for errors unless some validation / enforcement is put
in the way

**Comment by dasilvacontin** [comment]
Yeah, I'll try adding that to the precommit script. Maybe I can also check with CI?

**Comment by phillipj** [comment]
Precommit? This was a (pre)publish issue?

**Comment by dasilvacontin** [comment]
True, a `pre-push` hook would be better for this.

**Comment by phillipj** [comment]
FYI npm just got support for pre/postversion scripts. Might come in handy for altering the newly bumped version in e.g. mustache.js

**Comment by dasilvacontin** [comment]
Awesome! Looking forward to using it. :)
